### PR TITLE
i18n(fr): Update rss.mdx

### PR DESCRIPTION
**Pull request by pioupia**
#### What kind of changes does this PR include?

- Translated content

#### Description

- What does this PR change? Give us a brief description.
Updating the page `rss.mdx` in French.

### CODE REVIEW

**Comment by pioupia** [comment]
Needs to translate `markdown-content` page for the compatibility with this page.

File: src/pages/fr/guides/rss.mdx [hunk 1]
```diff
@@ -53,18 +69,19 @@ export const get = () => rss({
 
 Les `items` acceptent soit :
 
-1. [Un résultat de `import.meta.glob(...)`](#1-avec-un-résultat-importmetaglob) **(utilisez cette option pour les fichiers `.md` à l'interieur du répertoire `src/pages/`)**
+1. [Un résultat de `import.meta.glob(...)`](#1-avec-un-résultat-importmetaglob) **(utilisez cette option uniquement que pour les fichiers `.md` à l'intérieur du répertoire `src/pages/`)**
```
Comment (AndiKod) [inline src/pages/fr/guides/rss.mdx:72]: Hi,
The "que" in the string "uniquement que pour les fichiers" should be removed. 

It looks like a mix of the different ways to say the same thing, thus not a real expression.

**Review by AndiKod — CHANGES_REQUESTED** [review]
Hi,
Line 72 or the additions...
The "que" in the string "uniquement que pour les fichiers" should be removed.

It looks like a mix of the different ways to say the same thing, thus not a real expression.

**Review by AndiKod — APPROVED** [review]
I can't find anything misspelled or out of place in that commit, seems good enough.

**Comment by delucis** [comment]
This is ready to merge, right @pioupia?

**Review by AndiKod — APPROVED** [review]
No spelling errors in that commit.

**Review by delucis — APPROVED** [review]
Great! Thanks @pioupia and @AndiKod 🙌

**Pull request merged**